python_modules/libraries/dagster-dg-cli/dagster_dg_cli/cli/scaffold/branch/command.py

```diff
@@ -54,6 +58,17 @@ def is_prompt_valid_git_branch_name(prompt: str) -> bool:
     type=Path,
     help="Directory to write out session information for later analysis.",
 )
+@click.option(
+    "--diagnostics-level",
+    type=click.Choice(["off", "error", "info", "debug"]),
+    default="off",
+    help="Enable structured diagnostics logging at specified level (default: off).",
+)
+@click.option(
+    "--diagnostics-dir",
+    type=Path,
+    help="Directory to write diagnostics files (default: <system temp directory>/dg/diagnostics).",
+)
```

Comment:
does this conflict with the `--record` option? That also takes a directory to record the session. Diagnostics seems like it should replace this?

Reply:
yeah agree thanks will fix